sysutils/entr/Portfile

```diff
@@ -19,23 +21,11 @@ long_description    a utility for running arbitrary commands when \
 homepage            http://entrproject.org
 master_sites        ${homepage}/code
 
-checksums           rmd160  5abec400fdf589f9e3c4f576c835459480ed4e27 \
-                    sha256  c503c93ccffc1e6b2da979cdcb88c5a686e261103501eede7077fa089cdfef78 \
-                    size    25265
+checksums           rmd160  6e2c0f4828e2dc306185fc173658ba6ec44af468 \
+                    sha256  d6cbb3109d50d0d8e28fccd9843c522f896f00d4bfde7b1588cd01c1536afa21 \
+                    size    24674
 
-worksrcdir          eradman-entr-f4e2cbe57708
+build.pre_args-append       -f Makefile.macos
+destroot.pre_args-append    -f Makefile.macos
 
-# ./configure is just a simple batch file
-configure.pre_args
-configure.universal_args
-build.env-append    CC=${configure.cc} \
-                    "CFLAGS=${configure.cflags} [get_canonical_archflags cc]" \
-                    "LDFLAGS=${configure.ldflags} [get_canonical_archflags ld]"
-
-destroot.destdir    PREFIX="${destroot}${prefix}" \
-                    MANPREFIX="${destroot}${prefix}/share/man"
-
-test.run            yes
```

Comment:
are there no tests distributed anymore; it would be good to keep the test phase if possible?

Reply:
Good catch and thank you.  Tests re-enabled.